hackIDE/static/hackIDE/js/custom.js

```diff
@@ -123,6 +123,21 @@ $(document).ready(function(){
 		document.body.removeChild(element);
 	}
 
+	function downloadFile2(filename, text, lang) {
+
+		var ext = translateLangToExt(lang);
+		
+		var zip = new JSZip()
+		zip.file("test."+ext, text)
+		var xhr = new XMLHttpRequest();
+		xhr.onload = function() {
+			var downloaded = zip.generate({type : "blob"})
+			saveAs(downloaded, "test.zip")
+		}
+		xhr.open('get', 'https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js')
```

Comment:
Amazing! Everything looks fine except this line. Why do you need jQuery here? Can't you use the one that's already present locally in `js/` folder? @mananpal1997